Sources/Purchasing/Purchases/TransactionPoster.swift

```diff
@@ -112,6 +113,7 @@ final class TransactionPoster: TransactionPosterType {
                                          receipt: encodedReceipt,
                                          product: product,
                                          appTransaction: appTransaction,
+                                         metadata: data.metadata,
```

Comment:
Feel free to ignore since I don't know next to nothing about SDK. Why have a separate metadata parameter when data is already passed along in purchasedTransactionData?

Reply:
I think you are right... in fact, I believe we are not even using that parameter and should remove it. I'll check if that's true.

Reply:
Confirmed and removed unnecessary parameters. Great catch!!